### PR TITLE
feat(created an alert coverage gaps document): created a new doc for …

### PR DESCRIPTION
**Pull request by homelessbirds**
…alert coverage gaps

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - Alert coverage gaps
+metaDescription: Learn how alert coverage gaps in New Relic help you identify gaps in your alert system. 
+---
+import alertcoveragegaps from 'images/alert-coverage-gaps-homepage.png'
+
+import addanalert from 'images/add-an-alert.png'
+
+import createapolicyalertcoveragegaps from 'images/create-a-policy-coverage-gaps.png'
+
+Alert Coverage gaps highlight areas in your stack where you're missing observability coverage. In our Alerts and AI section [we've created a homepage](https://one.newrelic.com/nrai/detection-gaps/home) where you can see all your uncovered entities so you can assess what information you're team is lacking and address the knowledge gap.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:16]: i would imagine this likely should be sentence case, like most of our features, so more like 'Our alert coverage gaps feature highlights ....'

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+In an increasingly dynamic landscape, we know how important it is to stay on top of your stack and be proactively notified if anything goes wrong. But, as your team grows or changes, it can be hard to know if you're getting the alerts you need, when you need them. Alert coverage gaps gives your team a comprehensive list of all entities that your team *is not currenty covering*.  
+
+What does covering an entity mean? A covered entity means that you've set up an [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/) to notify you of how a particular entity is performing. For example, you might have created an alert condition that opens an incident if a host's CPU goes above 80%. An uncovered entity is part of your system that is unmonitored which means there could be unhealthy behavior that goes unchecked. 
+
+We created [alert coverage gaps](https://one.newrelic.com/nrai/detection-gaps/home) to highlight those areas of your stack where you might be missing valuable data so you can prevent incidents before they happen.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:22]: sentence seems a little hard to parse. maybe something like 'to highlight those areas of your stack where you might be missing valuable data. This data will help you ...'

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+What does covering an entity mean? A covered entity means that you've set up an [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/) to notify you of how a particular entity is performing. For example, you might have created an alert condition that opens an incident if a host's CPU goes above 80%. An uncovered entity is part of your system that is unmonitored which means there could be unhealthy behavior that goes unchecked. 
+
+We created [alert coverage gaps](https://one.newrelic.com/nrai/detection-gaps/home) to highlight those areas of your stack where you might be missing valuable data so you can prevent incidents before they happen. 
+
+## Find your coverage gaps [#find-your-coverage-gaps]
+
+On the Alerts and AI capabilites page, select alert coverage gaps in the naviagtion. There you will see a dashboard of all your entitites that are currently uncovered.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:26]: i think in general we can avoid 'capabilities' as a way to reference features/pages on the platform. reason being that it conflicts with our use of 'capabilities' in user mgmt space. So I'd just say 'On the **Alerts and AI** UI page...'

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+<figcaption>
+  When you select alert coverage gaps you will see a list of all your unmonitored entities.
+</figcaption>
+
+## Create an alert [#create-an-alert]
+
+Once you've selected an entity you'd like to monitor, then you can create an alert condition to notify your team of any abnormal behavior. 
+
+New Relic will dynamically suggest the conditions we believe should be monitored based on the selected entity and your specific system. Some [recommend alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/condition-recommendations/) we might be suggest are error percentage, apdex or response time.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:42]: some typos in this paragraph.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:16]: thanks!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:22]: thank you!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:26]: thanks!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:42]: thanks!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  title="Create an alert."
+  alt="A screenshot of creating an alert from the alert coverage gap dashboard"
+  src={addanalert}
+/>
+
+<figcaption>
+  Creating an alert for an unmonitored entity.
+</figcaption>
+
+Every alert condition needs to be routed through a [policy](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts) so you will be give the option to either select an existing policy or create a new one.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:56]:
```suggestion
Every alert condition needs to be routed through a [policy](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts) so you will be given the option to either select an existing policy or create a new one. 
```

Comment (zuluecho9) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:56]: wonder if 'routed through a policy' is optimal wording. Isn't it more true that conditions are attached to policies? Is there a way we can more clearly explain the connection? (but maybe I'm not up on the latest structure either)

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+Every alert condition needs to be routed through a [policy](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts) so you will be give the option to either select an existing policy or create a new one. 
+
+<img
+  title="Create a policy for alert coverage gaps."
+  alt="A screenshot of the alert coverage gap policy selection"
+  src={createapolicyalertcoveragegaps}
+/>
+
+<figcaption>
+  When you create an alert you will select which policy the alert will be grouped with.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:65]: related to above: 'which policy the alert will be grouped with': I wonder if that's explaining the structural connection between alert condition and policy well. we might want to be consistent with how we explain that connection (e.g., a policy contains conditions, or whatever)

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  - Alert coverage gaps
+metaDescription: Learn how alert coverage gaps in New Relic help you identify gaps in your alert system. 
+---
+import alertcoveragegaps from 'images/alert-coverage-gaps-homepage.png'
+
+import addanalert from 'images/add-an-alert.png'
+
+import createapolicyalertcoveragegaps from 'images/create-a-policy-coverage-gaps.png'
+
+Alert coverage gaps highlight areas in your stack where you're missing observability coverage. In our Alerts and AI section [we've created a homepage](https://one.newrelic.com/nrai/detection-gaps/home) where you can see all your uncovered entities so you can assess what information you're team is lacking and address the knowledge gap.
```
Comment (akane0915) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:16]: I would say `where you're missing alerts coverage`

Comment (akane0915) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:16]: Also, there is a typo.  Should be `your` team.  Not `you're` team.

Comment (akane0915) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:16]: I would say `address the coverage gap` rather than `address the knowledge gap`.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+In an increasingly dynamic landscape, we know how important it is to stay on top of your stack and be proactively notified if anything goes wrong. But, as your team grows or changes, it can be hard to know if you're getting the alerts you need, when you need them. Alert coverage gaps gives your team a comprehensive list of all entities that your team *is not currenty covering*.  
+
+What does covering an entity mean? A covered entity means that you've set up an [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/) to notify you of how a particular entity is performing. For example, you might have created an alert condition that opens an incident if a host's CPU goes above 80%. An uncovered entity is part of your system that is unmonitored which means there could be unhealthy behavior that goes unchecked. 
+
+We created [alert coverage gaps](https://one.newrelic.com/nrai/detection-gaps/home) to highlight those areas of your stack where you might be missing valuable data. This data will help you prevent incidents before they happen.
```
Comment (akane0915) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:22]: This paragraph seems repetitive of the one on line 16.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+<figcaption>
+  When you select alert coverage gaps you will see a list of all your unmonitored entities.
+</figcaption>
+
+## Create an alert [#create-an-alert]
+
+Once you've selected an entity you'd like to monitor, then you can create an alert condition to notify your team of any abnormal behavior. 
+
+New Relic will dynamically suggest the conditions we believe should be monitored based on the selected entity and your specific system. Some [recommend alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/condition-recommendations/) we might be suggest are error percentage, apdex, or response time.
```
Comment (akane0915) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:42]: Typo `We might be suggest`

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+  When you select alert coverage gaps you will see a list of all your unmonitored entities.
+</figcaption>
+
+## Create an alert [#create-an-alert]
+
+Once you've selected an entity you'd like to monitor, then you can create an alert condition to notify your team of any abnormal behavior. 
+
+New Relic will dynamically suggest the conditions we believe should be monitored based on the selected entity and your specific system. Some [recommend alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/condition-recommendations/) we might be suggest are error percentage, apdex or response time. 
+
+Once you select an alert condition you'd like to monitor, then your team will be alerted when this signal deviates from its normal baseline, upper and lower, for at least 5 minutes by 3.00 standard deviation(s).
```
[the remark this answers is not in the document]
Comment (akane0915) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:44]: I would not say the details `when this signal deviates from its normal baseline, upper and lower, for at least 5 minutes by 3.00 standard deviations.`. The users can modify the alert before saving so they could change all of these attributes if they wanted to.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 5 above)
Comment (akane0915) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:56]: Typo `you will be give the option`

Comment (akane0915) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx:56]: All conditions are associated with a policy.

**Review by akane0915 — CHANGES_REQUESTED** [review]

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 7 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx]: Thanks! I fixed this.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx (hunk 9 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx]: Thank you!
